<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-UBQJ9QQ" ignored="0" name="First Test" tests="9" failures="2" timestamp="01 Jun 2020 08:51:06 GMT" time="0.064" errors="0">
  <testcase name="testLogin" time="0.01" classname="com.Sample.A"/>
  <testcase name="testLogin" time="0.002" classname="com.Sample.B">
    <skipped/>
  </testcase> <!-- testLogin -->
  <testcase name="testLogin" time="0.001" classname="com.Sample.B">
    <skipped/>
  </testcase> <!-- testLogin -->
  <testcase name="testLogin" time="0.0" classname="com.Sample.B">
    <skipped/>
  </testcase> <!-- testLogin -->
  <testcase name="testLogin" time="0.001" classname="com.Sample.B">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.Sample.B.testLogin(B.java:10)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- testLogin -->
  <testcase name="testLogin" time="0.001" classname="com.Sample.C">
    <skipped/>
  </testcase> <!-- testLogin -->
  <testcase name="testLogin" time="0.002" classname="com.Sample.C">
    <skipped/>
  </testcase> <!-- testLogin -->
  <testcase name="testLogin" time="0.001" classname="com.Sample.C">
    <skipped/>
  </testcase> <!-- testLogin -->
  <testcase name="testLogin" time="0.001" classname="com.Sample.C">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.Sample.C.testLogin(C.java:10)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- testLogin -->
</testsuite> <!-- First Test -->
